gulpfile.js

```diff
@@ -108,11 +108,11 @@ gulp.task('4:jsBundle', function(){
 			var comment = '/*! ' + filename + ' */ \n';
 			return comment + contents;
 		}))
+        .pipe(jshint())
```

Comment:
JS linting has been moved up to take place before stripping empty lines and comments, so that the errors reported correspond with the correct line numbers in the source code.